src/gridstack.js

```diff
@@ -945,15 +982,55 @@
         });
     };
 
-    GridStack.prototype.cell_height = function(val) {
+    function parseHeight(val) {
+        var height = val;
+        var height_unit = 'px';
+        if (height && _.isString(height)) {
+            var match = height.match(/^([0-9]+)(px|em|rem)?$/);
+            if (!match) {
+                throw new Error('Invalid height');
+            }
+            height_unit = match[2];
+            height = parseInt(match[1]);
+        }
+        return {height: height, unit: height_unit};
+    }
+
+    GridStack.prototype.vertical_margin = function (val, noUpdate) {
+        if (typeof val == 'undefined') {
+            return this.opts.vertical_margin;
+        }
+
+        var heightData = parseHeight(val);
+
+        if (this.opts.vertical_margin_unit === heightData.unit && this.opts.height === heightData.height) {
+            return ;
+        }
+        this.opts.vertical_margin_unit = heightData.unit;
+        this.opts.vertical_margin = heightData.height;
+
+        !noUpdate && this._update_styles();
```

Comment:
`!noUpdate && this._update_styles();`
Please, no. We can definitely bear one more line to increase readability. ;)